Generics.md

```diff
@@ -9,23 +9,23 @@ If adding a new document, you will need to update the defconf variable by either
 
 	(defconfig approved-pipeline-documents
   		{:default {:grid ["0.0.1"]
-             :dataqualitysummary ["1.0.0"]
-             :orderoption ["1.0.0"]
-             :serviceentry ["1.0.0"]
-             :serviceoption ["1.0.0"]}
+             :data-quality-summary ["1.0.0"]
+             :order-option ["1.0.0"]
+             :service-entry ["1.0.0"]
+             :service-option ["1.0.0"]}
```

Comment:
Can you look in Kibana and tell me what the index names are?  Are they 1_generic_dataqualitysummary
 or 1_generic_data-quality-summary....
If it is the latter I want to talk with either Jay or Leo about deleting the existing indexes before adding these and if we do that then we can use the old toggle and not need the new one, although i think the new one would be good anyway.

Reply:
so to answer your question more directly, yes the index names will be different. (just with all underscores)

Reply:
Ok, I am happy, now.  Just talked with @ygliuvt and since no one is using them yet, we can use the new names and delete the old indices.  Then we can use bulk update to update anything that has been created.